### PR TITLE
[[: add German translation

### PR DESCRIPTION
**Pull request by marvinborner**
Basically slightly modified `[.md`

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
Welcome to tldr @marvinborner. Thanks for your contribution.

File: pages.de/common/[[.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# [[
+
+> Teste und vergleiche Dateien und Variablen.
```
Comment (pixelcmtd) [inline pages.de/common/[[.md:3]:
```suggestion
> Teste Dateitypen und vergleiche Werte.
```

File: pages.de/common/[[.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+- Überprüfe, ob eine bestimmte Variable gleich oder ungleich einem bestimmen String ist:
+
+`[[ ${{variable}} {{==|!=}} "{{string}}" ]]`
+
+- Überprüfe, ob ein bestimmter String einem bestimmten regulären Ausdruck oder glob-Muster entspricht:
+
+`[[ ${{variable}} {{==|=~}} {{muster}} ]]`
+
+- Überprüfe, ob eine Variable gleich, ungleich, größer, kleiner, größer-gleich oder kleiner-gleich als eine bestimme Zahl ist:
```
Comment (pixelcmtd) [inline pages.de/common/[[.md:15]:
```suggestion
- Überprüfe, ob eine Variable gleich/ungleich/größer/kleiner/größer-gleich oder kleiner-gleich als eine bestimme Zahl ist:
```

Comment (marvinborner) [reply pages.de/common/[[.md:15]: Yep, I agree to both of your changes. As that's been copied from `[.md`, we'd need to change that too, though. What's the procedure for that? Should I create a new PR?

**Review by pixelcmtd — APPROVED** [review]

File: pages.de/common/[[.md (hunk 2 above)
Comment (pixelcmtd) [reply pages.de/common/[[.md]: I'd suggest you do.

**Pull request merged**